qlty-config/src/config/issue_transformer.rs

```diff
@@ -1,9 +1,21 @@
 use qlty_types::analysis::v1::Issue;
+use rayon::prelude::*;
 use std::fmt::Debug;
 
 pub trait IssueTransformer: Debug + Send + Sync + 'static {
     fn initialize(&self) {}
-    fn transform(&self, issue: Issue) -> Option<Issue>;
+
+    fn transform(&self, issue: Issue) -> Option<Issue> {
+        Some(issue)
+    }
+
+    fn transform_batch(&self, issues: Vec<Issue>) -> Vec<Issue> {
+        issues
+            .par_iter()
+            .filter_map(|issue| self.transform(issue.clone()))
```

Comment:
I don't think we should need to clone an individual issue in order to transform it.

I'm not sure if it'll work with the filter_map construct exactly the same, but the previous code avoided the clone of each issue.

Reply:
The previous code used `.cloned()` which cloned all elements implicitly, so this is the same result. We could use `.cloned()` here as well, I'm impartial.